geniza/common/metadata_export.py

```diff
@@ -132,8 +133,12 @@ def iter_csv(self, fn=None, pseudo_buffer=False):
             else Echo()
         ) as of:
             writer = csv.DictWriter(
-                of, fieldnames=self.csv_fields, extrasaction="ignore"
+                of,
+                fieldnames=self.csv_fields,
+                extrasaction="ignore",
             )
+            # start with byte-order mark so Excel will read unicode properly
+            yield codecs.BOM_UTF8
```

Comment:
Nice! huh, never seen this problem or solution before

Reply:
I've run into it before, which is why I recognized it immediately when I finally figured out why Ksenia and I were seeing different results.

Reply:
In short (see my comment below) I think all we need to do to make this UTF-8 character yield statement work for both non-http and http-streaming exports, we just need to add a `of.write(codecs.BOM_UTF8)` above or below this line, so that it's both written to the file buffer and yielded out for anyone iterating over the written-out strings (i.e. the http streamer).

Reply:
Thank you for commenting and explaining all this — I thought that I wasn't getting it to work but I was looking at the wrong export file 🤦‍♀️ 

I've confirmed that we're getting one (and only one) BOM for both csv file on disk and http stream, so I think we're good on this one now